apps/sms/services/js/moz_settings/moz_settings_shim.js

```diff
@@ -0,0 +1,65 @@
+/* -*- Mode: js; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- /
+/* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
+
+/* global BridgeServiceMixin */
+
+/* exported MozSettingsShim */
+
+(function(exports) {
+'use strict';
+
+/**
+ * Name of the service for mozSettings API shim.
+ * @type {string}
+ */
+const SERVICE_NAME = 'moz-settings-shim';
+
+/**
+ * Array of method names that need to be exposed for API shim.
+ * @type {Array.<string>}
+ */
+const METHODS = Object.freeze(['get', 'set']);
+
+var mozSettings = null;
+
+var MozSettingsShim = {
+  init(settings) {
+    if (!settings) {
+      return;
+    }
+    mozSettings = settings;
+
+    this.initService();
+  },
+
+  /* Methods */
+
+  /**
+   * Shim for mozSettings.createLock().get API.
+   * @param {String} key Key name for Settings.
+   * @returns {Promise} Promise that return the settings value from settings DB.
+   */
+  get(key) {
```

Comment:
> If it's really not a performance-sensitive case, maybe we can wrap this in settings service or other client side since it's simply API shim here.

I agree, it's more client or service (that's one more abstraction level) privilege to enhance API :) I'd do this later only if we see performance penalty for multiple requests.